projects/NETCore.md

```diff
@@ -1,22 +1,22 @@
 # .NET Core
 
-.NET Core is the small optimized runtime that is the basis of [ASP.NET Core 5](http://www.dotnetfoundation.org/aspnet-5). It currently runs on Windows, and will be extended to support Linux and Mac. It is a high-performance and modular design, and supports full side by side to make it easy to adopt new .NET Core versions without affecting other apps. These products are actively developed by the .NET team and in collaboration with a community of open source developers. Together we are dedicated to improving and extending the .NET platform with new features and for new scenarios.
+.NET Core is the small optimized runtime that is the basis of [ASP.NET Core](http://www.dotnetfoundation.org/aspnet-core). It currently runs on Linux, Mac, and Windows. It is a high-performance and modular design, and supports full side by side to make it easy to adopt new .NET Core versions without affecting other apps. These products are actively developed by the .NET team and in collaboration with a community of open source developers. Together we are dedicated to improving and extending the .NET platform with new features and for new scenarios.
```

Comment:
This text is now a bit wrong. It should say:

- .NET Core is blah.
- ASP.NET Core builds on top of it.